master/docs/manual/installation.rst

```diff
@@ -29,6 +29,9 @@ Python: http://www.python.org
   Buildbot requires Python-2.6 or later on the master, although Python-2.7 is recommended.
   The slave runs on Python-2.5.
 
+  Note that this should be a "normal" build of Python.
```

Comment:
It produces extra output on stdout, which confuses tests that examine that stdout.